src/marqo/core/structured_vespa_index/structured_vespa_index.py

```diff
@@ -453,6 +453,49 @@ def _get_tensor_search_term(self, marqo_query: MarqoTensorQuery) -> str:
         else:
             return ''
 
+    def convert_to_in_list_str(self, value_list: list, marqo_field_name: str, marqo_field_type: FieldType) -> str:
```

Comment:
You may also want to move this function under `_get_filter_term()`, like `escape`. Ask Farshid to confirm on this.

Reply:
Done